lib/main.dart

```diff
@@ -180,6 +177,7 @@ Future<void> main() async {
     optionThemeTrueBlack: false,
     optionThemeColorScheme: 'aquaBlue',
     optionTrendsLocation: jsonEncode({'name': 'Worldwide', 'woeid': 1}),
+    optionImproveNonConfirmationBias: false,
```

Comment:
`optionNonConfirmationBiasMode` fits better, as it's more 1:1 with the name of the preference.